app/typer/sanity/sanity.ts

```diff
@@ -33,3 +33,9 @@ export interface ITekstinnhold {
 export enum ESanityFlettefeltverdi {
   SØKER_NAVN = 'SØKER_NAVN',
 }
+
+export interface AppContext {
+  sanityTekster: ITekstinnhold;
+  språk: [LocaleType, React.Dispatch<React.SetStateAction<LocaleType>>];
+  fornavn: string;
```

Comment:
Ser for meg at vi kommer til å bruke mer av brukerens data enn fornavnet, så tenker vi kan putte all dataen vi får fra backend inn i contexten

Reply:
Det ser vi også for oss, så la opp til det nå her [0c783c0](https://github.com/bekk/nav-familie-endringsmelding/pull/20/commits/0c783c0a196753362b3209b97151356c96c0e1b6)

Reply:
Med en liten fiks her [8bcba91](https://github.com/bekk/nav-familie-endringsmelding/pull/20/commits/8bcba91dd0ae7f469843e71b5ae2f183293d3e24)